python-client/giskard/scanner/robustness/unit_perturbation.py

```diff
@@ -0,0 +1,196 @@
+import pandas as pd
+from giskard.ml_worker.testing.registry.transformation_function import transformation_function
+import random
+from transformers import AutoTokenizer, AutoModelForTokenClassification
+from transformers import pipeline
+from giskard.scanner.robustness.entity_swap import masculine_to_feminine, feminine_to_masculine, minority_groups, religion_dict
+
+
+class TransformationGenerator:
+    def __init__(self, model, dataset):
+        self.model = model
+        self.dataset = dataset
+        self.column_types = dataset.column_types
+        self.text_perturbation_generator = TextTransformer()
+
+    def generate_std_transformation(self, feature):
+        mad = self.dataset.df[feature].mad()
+
+        @transformation_function()
+        def func(x: pd.Series) -> pd.Series:
+            if self.column_types[feature] == "numeric":
+                x[feature] += 3 * mad
+                return x
+            return x
+
+        return func
+
+    def text_transformation(self, feature):
+        @transformation_function()
+        def func(x: pd.Series) -> pd.Series:
+            if self.column_types[feature] == 'text':
+                self.text_perturbation_generator.load(x[feature])
+                x[feature] = self.text_perturbation_generator.execute_protocol()
+                return x
+
+        return func
+
+
+class TextTransformer:
+    def __init__(self):
+        self.text = None
+        self.tokenizer = AutoTokenizer.from_pretrained("dslim/bert-base-NER")
+        self.model = AutoModelForTokenClassification.from_pretrained("dslim/bert-base-NER")
```

Comment:
This is an English language model

Reply:
I'll change it with https://huggingface.co/Davlan/distilbert-base-multilingual-cased-ner-hrl